qubesmanager/qvm_template_gui.py

```diff
@@ -184,7 +186,10 @@ def description(self) -> str:
             text += tr("<b>Template upgrade available: </b>")
             if self.template_status == 'upgradable':
                 text += tr("yes, newest available version: " +
-                           self.newest_available_version + "<br>")
+                           self.newest_available_version)
```

Comment:
It was an issue before, but variable part should not be under the first arg to `tr()`

Reply:
fixed